### DISCUSSION TITLE
Fall Guys compatibility

### DISCUSSION THREAD

**Discussion by Ahmed-E-86** [Heroic-Games-Launcher/HeroicGamesLauncher#1509] [Q&A]
Does Fall Guys run through HeroicGamesLauncher on Linux?

**Comment by sheabert33** [comment]
I've been able to get the main menu to load but I'm unable to start a game.

**Comment by Etaash-mathamsetty** [comment]
download the latest version (beta 2 rn), enable the EAC runtime toggle and enable the EOS overlay, then apply the workarounds that are needed for the game (editing the ini file and copying/symlinking the .so file)
